css/index.scss

```diff
@@ -12,8 +12,9 @@
   margin-top: -$nav-height;
   padding: $nav-height $splash-horizontal-padding 0;
 
-  // Set minimum height to fill all the space but the footer.
+  // Set height to fill all the space but the footer.
   min-height: 100vh;
```

Comment:
I don't think CSS allows for multiple `min-height` properties, so we probably want to remove this one. `1vh` is provided as a default value in `var`.

Otherwise, the change looks good. The debounced calculation at least makes the height change not terrible for the user.

Reply:
You're right, it doesn't. This line is a fallback in case the next line doesn't work in certain browsers. We can still remove it if you'd like.

Reply:
Good point. I'm happy to have a fallback for compatibility.